templates/block/block--system-menu-block.html.twig

```diff
@@ -0,0 +1,76 @@
+ * - configuration: A list of the block's configuration values.
+ *   - label: The configured label for the block.
+ *   - label_display: The display settings for the label.
+ *   - provider: The module or other provider that provided this block plugin.
+ *   - Block plugin specific settings will also be stored here.
+ * - content: The content of this block.
+ * - attributes: HTML attributes for the containing element.
+ *   - id: A valid HTML ID and guaranteed unique.
+ * - title_attributes: HTML attributes for the title element.
+ * - content_attributes: HTML attributes for the content element.
+ * - title_prefix: Additional output populated by modules, intended to be
+ *   displayed in front of the main title tag that appears in the template.
+ * - title_suffix: Additional output populated by modules, intended to be
+ *   displayed after the main title tag that appears in the template.
+ *
+ * Headings should be used on navigation menus that consistently appear on
+ * multiple pages. When this menu block's label is configured to not be
+ * displayed, it is automatically made invisible using the 'visually-hidden' CSS
+ * class, which still keeps it visible for screen-readers and assistive
+ * technology. Headings allow screen-reader and keyboard only users to navigate
+ * to or skip the links.
+ * See http://juicystudio.com/article/screen-readers-display-none.php and
+ * http://www.w3.org/TR/WCAG-TECHS/H42.html for more information.
+ */
+#}
+{%
+  set classes = [
+    'block',
+    'block-menu',
+    'navigation',
+    'menu--' ~ derivative_plugin_id|clean_class,
+  ]
+%}
+{% set heading_id = attributes.id ~ '-menu'|clean_id %}
+
+<div class="navbar js-nav-expandable">
+  <div class="navbar__container">
+    <div class="navbar__inner">
+      <button class="navbar__button js-nav-mobile-closable">
+        <span class="navbar__text visually-hidden">menu</span>
+        <svg class="i-svg i-svg--menu"><use xlink:href="#i-menu" xmlns:xlink="http://www.w3.org/1999/xlink"></use></svg>
+        <svg class="i-svg i-svg--close"><use xlink:href="#i-close" xmlns:xlink="http://www.w3.org/1999/xlink"></use></svg>
+      </button>
+
+      <div class="navbar__group js-nav-group-viewable js-nav-item-expandable">
+        <nav role="navigation" aria-labelledby="{{ heading_id }}"{{ attributes.addClass(classes)|without('role', 'aria-labelledby') }}>
+          {# Label. If not displayed, we still provide it for screen readers. #}
+          {% if not configuration.label_display %}
+            {% set title_attributes = title_attributes.addClass('visually-hidden') %}
+          {% endif %}
+          {{ title_prefix }}
+          <h2{{ title_attributes.setAttribute('id', heading_id) }}>{{ configuration.label }}</h2>
+          {{ title_suffix }}
+
+          {# Menu. #}
+          {% block content %}
+            {{ content }}
+          {% endblock %}
+        </nav>
+        <div class="navbar__search">
+          <form class="search-toggle">
```

Comment:
I'm going to need to find out where this will need to submit to and how. Perhaps another ticket for this item.